.pipeline/config.yml

```diff
@@ -11,3 +11,10 @@ steps:
     owner: 'SAP'
     repository: 'jenkins-library'
     releaseBodyHeader: ''
+  githubCreatePullRequest:
```

Comment:
Can we add reviewer?

Reply:
I have only seen assignees in the step options, and I'm not sure if and how we can add the `SAP/jenkins-library-team` there.

Reply:
Maybe we should add *CODEOWNERS* to cover this..

Reply:
Not sure if that is my preferred solution. I would do it iteratively. Merge this one (smaller scope of change), see how it works out and either add the feature to the step to add reviewers, or assign the team or some people to it later.

Reply:
Sure, PR scope is fine from my side.